mimic.py

```diff
@@ -47,16 +47,34 @@
 import sys
 
 
+def file_word_list(file):
+    with open(file, 'r') as file:
+        words_list = file.read().lower().split()
+    return words_list
+
+
 def mimic_dict(filename):
```

Comment:
I feel like this is doing more work than it needs to which would explain why your run time is longer than it needs to be.  It functions fine, but it should really hang when you run it.

Reply:
This being mimic_dict(), I jacked up my comment placement.